src/warnet/utils.py

```diff
@@ -515,3 +515,9 @@ def convert_unsupported_attributes(graph):
                 continue
             else:
                 edge_data[key] = str(value)
+
+
+def generate_as():
+    while True:
+        as_number = random.randint(1, 64496)  # I think these are not "reserved"
```

Comment:
magic number alert! do we use this value anywhere else? even if not, might be good to set it as a global const with a nice comment explaining what the number means